packages/cli/src/ocommands/build.js

```diff
@@ -0,0 +1,43 @@
+const { flags: Flags } = require('@oclif/command');
+
+const BaseCommand = require('../baseCommand');
+const buildFlags = require('../buildFlags');
+const { build } = require('../utils/build');
+const { BUILD_PATH, SOURCE_PATH } = require('../constants');
+
+const dumbPathsFlag = 'disable-dependency-detection';
+const mapFlag = 'include-js-map';
+
+class BuildCommand extends BaseCommand {
+  async run() {
+    this.flags = this.parse(BuildCommand).flags;
+    // flags['disable-dependecy-injection'] is ugly, so let's make the opts better for programming
+    // alternatively, rename the flag to something without dashes. camelCase works fine
+    const opts = {
+      dumbPaths: Boolean(this.flags[dumbPathsFlag]),
+      includeMaps: Boolean(this.flags[mapFlag])
+    };
+
+    await build(opts);
+
+    this.log(
+      `\nBuild complete! Moved ${BUILD_PATH} and ${SOURCE_PATH} ! Try the \`zapier upload\` command now.`
+    );
+  }
+}
+
+BuildCommand.rawFlags = {
```

Comment:
`Command.flags` is the standard place to store flags, but if we want to share them across commands (like build + push sharing the same ones), then we need to isolate `build`'s flags (before it's mixed with `debug`, etc).